src/data/router.jsx

```diff
@@ -9,11 +9,14 @@ const SeeProject = lazy(() => import("../pages/SeeProject"));
 const ProjectTask = lazy(() => import("../pages/ProjectTask"));
 const Notification = lazy(() => import("../pages/NotificationPage"));
 const Login = lazy(() => import("../pages/Login"));
+const Signup = lazy(() => import("../pages/Signup"));
+const Error = lazy(() => import("../pages/Error"));
```

Comment:
You would have added this error component by opening a separate issue and creating a new branch for that. Please remember this for future commits. 😊